src/main/java/org/jenkinsci/plugins/fodupload/models/DastScanJobModel.java

```diff
@@ -30,8 +30,23 @@ public class DastScanJobModel {
     String entitlementFrequencyId;
     String entitlementFrequencyType;
     String userSelectedEntitlement;
-
     String networkAuthType;
+    String selectedApi;
```

Comment:
Can we have one more Overload Constructor to cover for API so We don't need to invoke Ctor with null values passed for website /workflow driven

Reply:
Sure, will do it in another PR